feconf.py

```diff
@@ -284,7 +284,8 @@
 # An array containing the accepted audio extensions for uploaded files and
 # the corresponding MIME types.
 ACCEPTED_AUDIO_EXTENSIONS = {
-    'mp3': ['audio/mp3']
+    'mp3': ['audio/mp3'],
+    'flac': ['audio/flac']
```

Comment:
This is a test PR, it should not be changing the functionality of the code.

Can you not change this? For now, having an assumption that all files are .mp3 makes it easier to reason about other processes. In the future we may support other formats but this PR is not the place for that.

Reply:
Without adding the flac extension `AudioUploadHandler` will only work for mp3 files and [this line](https://github.com/oppia/oppia/blob/develop/core/controllers/translator.py#L95) will remain unused. Should I remove this line then? Also if the flac files are not supported should instances of flac files and their associated tests be removed?

Reply:
You can swap feconf.ACCEPTED_AUDIO_EXTENSIONS just for the purposes of the test.

Reply:
Done!